### ISSUE TITLE
Envio de imagem não assina nome do operador

### ISSUE THREAD

**Issue by rapososoftwares** [ticketz-oss/ticketz#212]
Ao enviar imagem, o sistema não assina o nome do operador

![image](https://github.com/user-attachments/assets/557ff27e-4908-4593-8ad4-740d50913e09)

Seria interessante assinar, assim como é feito nas msgs, um exemplo disso

![image](https://github.com/user-attachments/assets/b4daf84d-e599-4493-a34b-0be9f79e1c4b)

**Comment by allgood** [comment]
Marquei como "investigate" porque a assinatura é mais algo a ser indicado ao contato do que ao administrador do Ticketz. Hoje todas as mensagens enviadas no Ticketz já registram o usuário no banco de dados e talvez o ajuste seja fazer a exibição desse nome pelo menos quando passar por cima da mensagem enviada.

**Comment by rapososoftwares** [comment]
Sim, mas o contato não recebe a assinatura do atendente quando é enviado imagem, apenas texto
